plugins/openai_plugin.py

```diff
@@ -0,0 +1,229 @@
+import json
+import logging
+import time
+
+import requests
+import urllib3
+
+from plugins import plugin
+from result import RequestResult
+
+urllib3.disable_warnings()
+"""
+Example plugin config.yaml:
+
+plugin: "openai_plugin"
+plugin_options:
+  streaming: True/False
+  host: "http://127.0.0.1:5000/v1/completions"
```

Comment:
what about  v1/chat/completions ?

Reply:
Hmm, maybe I should expose this as an option, so we can hit the chat endpoint or the completions endpoint.